src/__tests__/react.test.tsx

```diff
@@ -190,3 +191,54 @@ test('ReactTinyMarkup custom renderers', () => {
     )
   ).toEqual('innerinvi<bbb>s</bbb>ible<bbb>left in</bbb>');
 });
+
+test('ReactTinyMarkup attributes', () => {
+  const source =
+    '<b class="abc" href="nope.com">show class but not disabled</b>';
+  const expected = '<b class="abc">show class but not disabled</b>';
+  expect(
+    ReactDOMServer.renderToStaticMarkup(
+      <ReactTinyMarkup allowedAttributes={{ class: 'className' }}>
+        {source}
+      </ReactTinyMarkup>
+    )
+  ).toEqual(expected);
+});
+
+test('ReactTinyMarkup attributes with custom renderer', () => {
+  const source = `<b class="abc" href="nope.com">show class but not disabled</b><img alt="none" src="yep.com"/><img  src="nope.com"/>`;
+  const expected = `<bbb class="abc" title="b">show class but not disabled</bbb><img alt="NONE" src="yep.com"/><img src="nope.com" alt=""/>`;
+
+  expect(
+    ReactDOMServer.renderToStaticMarkup(
+      <ReactTinyMarkup
+        allowedAttributes={{ class: 'className', src: 'src', alt: 'alt' }}
```

Comment:
For same name attrs we just need to add the key:
```suggestion
        allowedAttributes={{ class: 'className', src: '', alt: '' }}
```